src/app/login/login.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6 d-flex align-items-center justify-content-center">
+
+      <form [formGroup]="logForm" (ngSubmit)="onSubmit()">
+
+        <div class="text-center">
+          <h2>LOGIN</h2>
+          <p class="text-secondary">Please enter your credentials</p>
+        </div>
+
+        <div class="p-1">
+          <div class="error-message" *ngIf="submitted">
```

Comment:
error message should appear when the form is submitted **and has an error**

Reply:
The error message already will not appear until the user clicks submit and has an error,
in addition of that, if he did not enter a correct username or password it will show a red border + shadow 
trying to give the user a hint to fix them